alf/utils/common.py

```diff
@@ -441,7 +442,7 @@ def forward(self, observation):
         """Normalize a given observation. If during unroll, then first update
         the normalizer. The normalizer won't be updated in other circumstances.
         """
-        if not is_training():
+        if is_training():
```

Comment:
Probably it's better to add an option for the user to select when to update statistics

Reply:
For unroll, we need to do it with HER sampling, which means we probably cannot reuse this class.

For the non-her case, we also need to fix the case where test exp should be excluded.

Added a TODO.

Reply:
Might just set the flag to be one of "train", "unroll", and "eval"?